conf/squid/common/ms_2way_5node_1client_rh.yaml

```diff
@@ -0,0 +1,94 @@
+globals:
+  - ceph-cluster:
+      name: ceph-pri
+      node1:
+        role:
+          - _admin
+          - installer
+          - mon
+          - mgr
+          - crash
+      node2:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node3:
+        role:
+          - mon
+          - mgr
+          - crash
+          - rgw
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node4:
+        role:
+          - mon
+          - mgr
+          - crash
+          - osd
+        no-of-volumes: 6
+        disk-size: 40
+      node5:
```

Comment:
please add `rbd-mirror` as well in node5, refer here
https://github.com/red-hat-storage/cephci/blob/main/conf/squid/rbd/5-node-2-clusters.yaml#L62

Reply:
thanks @sunilangadi2 for the reference. I have added it.